packages/pinia/__tests__/store.spec.ts

```diff
@@ -366,4 +366,17 @@ describe('Store', () => {
     useMyStore()
     expect(warnTextCheckPlainObject('poInit')).toHaveBeenWarnedTimes(0)
   })
+
+  it('only warns when state name conflicts with getters name', () => {
+    const useStore = defineStore({
+      id: 'main',
```

Comment:
Pass it as the first argument instead

Reply:
What is the difference between passing store name(id) as the first argument and using it as property?
Other tests weren't using id as the first argument!